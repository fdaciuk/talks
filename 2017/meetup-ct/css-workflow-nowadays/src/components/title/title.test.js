'use strict'

import React from 'react'
import renderer from 'react-test-renderer'
import Title from './index'

it('Should Title match to snapshot', () => {
  const tree = renderer.create(<Title>TÃ­tulo</Title>).toJSON()
  expect(tree).toMatchSnapshot()
})
