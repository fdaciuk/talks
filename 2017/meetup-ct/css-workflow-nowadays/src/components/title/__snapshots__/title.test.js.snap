// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Title match to snapshot 1`] = `
<h1>
  TÃ­tulo
</h1>
`;
